tests/test_loganalyzer.py

```diff
@@ -0,0 +1,68 @@
+"""

```

Comment:
README.md in 'tests/loganalyzer/' with usage description would be better. User can run all tests including this one.